script/1_0_2current_en_script/Ciel/Day 5/05_01_CIEL05_1DASH.txt

```diff
@@ -137,5 +137,5 @@ while keeping an eye on the duo's movements.//shifted the punctuations
 "...I'm sorry. I heard that you can get a part-time job here..."
 I said in a meek voice while looking closely at the man named Toshi.
 <Page42>
-"...A part-time job, huh? Where did you hear about it? It's been I've only been talking about it to a bunch of stupid girls lately."
-"I got to know it by hearsay. That you could get a lot of money by spending the night of the apartment at the north gate."
\ No newline at end of file
+"...A part-time job, huh? Where did you hear about it? It's something that I've only talked about with a bunch of stupid girls lately?"
+"I got to know it by hearsay. That you could get a lot of money by spending the night at the apartment near the north gate."
```

Comment:
"I just heard a rumour. Supposedly, you can get a lot of money by spending the night at the apartment near the north gate."